src/legacy/core_plugins/kibana/public/dashboard/dashboard_app_controller.tsx

```diff
@@ -547,11 +547,13 @@ export class DashboardAppController {
 
     $scope.showAddPanel = () => {
       dashboardStateManager.setFullScreenMode(false);
-      $scope.kbnTopNav.click(TopNavIds.ADD);
+      // Temp solution for triggering menu click.
+      navActions[TopNavIds.ADD](null, null, null);
```

Comment:
> // Temp solution for triggering menu click.

How will we know when this can be removed? (Or I guess I'm asking "temporary until when?")

Reply:
Improved comment: 
      /* 
      * Temp solution for triggering menu click.
      * When de-angularizing this code, please call the underlaying action function 
      * directly and not via the top nav object. 
      **/